trento/xml/article_sap_trento.xml

```diff
@@ -256,29 +271,36 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       Trento is powered by event driven technology. Registered events are stored in a  &postgresql; database with a retention
       period of 10 days. For each host registered with Trento, you need to allocate at least 1.5GB of space in the &postgresql; database.
      </para>
-      <para> While the &t.server; supports various usage scenarios,
-        depending on the existing infrastructure, it is designed to be
-        cloud native and OS agnostic. It can
-        be installed on the following services: </para>
+          <para>
+      Prometheus is used by Trento Server to collect metric data collected by the exporters in the agent hosts.
+           It is an optional component and only required if we want to have graphical charts of the collected metrics 
+           in the console. Otherwise we can disable the charts upon installation. We can perform an ad-hoc installation of
+           Prometheus for Trento or use an existing one. In the case of a systemd deployment on SLES for SAP SP3, only the latter is possible.
+     </para>
+      <para> For a &k8s; deployment, the follwing services can
+        be useds: </para>
       <!--
         toms 2021-12-06: taken from "Draft - Various Tidbits regarding Productization"
         (https://confluence.suse.com/x/tgCnMg)
       -->
       <itemizedlist>
+               <listitem>
+          <para>K3s</para>
+        </listitem>
         <listitem>
           <para>RKE1 (&rancher.k8s.engine; version 1)</para>
         </listitem>
         <listitem>
           <para>RKE2</para>
         </listitem>
         <listitem>
-          <para>any other CNCF-certified &k8s; running on x86_64 architecture</para>
+          <para>any other CNCF-certified &k8s; running on x86_64 architecture. This includes AKS in Azure, EKS in AWS and GKE in GCP.</para>
         </listitem>
       </itemizedlist>
-     <para> A proper, production ready installation of &t.server; requires &k8s;
-      knowledge. The Helm chart is meant to be used by customers lacking such knowledge
-      or who want to get started quickly. However, Helm chart delivers a basic deployment of the &t.server; with all the components running
-      on a single node of the cluster.</para>
+     <para> A proper, production ready installation of &t.server; on &k8s; requires technical
+      knowledge of this platform. The Helm chart is meant to be used by customers lacking such knowledge
+      or who want to get started quickly. But bear in mind that the Helm chart delivers a basic deployment of the &t.server; with all the components running
+      on a single node of the cluster unless adapted otherwise by the customer.</para>
```

Comment:
First you start with first person and the sentence ends with "by the customer." I think you can remove the latter (or use "by you"):

```suggestion
      or who want to get started quickly. But bear in mind that the Helm chart delivers a basic deployment of the &t.server; with all the components running
      on a single node of the cluster unless adapted otherwise.</para>
```